lbrynet/wallet/ledger.py

```diff
@@ -51,36 +50,20 @@ def __init__(self, *args, **kwargs):
 
     @property
     def resolver(self):
-        return Resolver(self.headers.claim_trie_root, self.headers.height, self.transaction_class,
-                        hash160_to_address=self.hash160_to_address, network=self.network, ledger=self)
-
-    async def resolve(self, page, page_size, *uris):
-        for uri in uris:
-            try:
-                parsed_uri = parse_lbry_uri(uri)
-                if parsed_uri.claim_id:
-                    validate_claim_id(parsed_uri.claim_id)
-            except URIParseError as err:
-                return {'error': err.args[0]}
-            except Exception as e:
-                return {'error': str(e)}
-        try:
-            resolver = self.resolver
-            resolutions = await self.network.get_values_for_uris(self.headers.hash().decode(), *uris)
-            return await resolver._handle_resolutions(resolutions, uris, page, page_size)
-        except Exception as e:
-            log.exception(e)
-            return {'error': str(e)}
+        return Resolver(self.headers, self.transaction_class, network=self.network, ledger=self)
```

Comment:
Since you're passing in the `ledger` (which has everything on it), do you need any of the other arguments? Seems like this entire call can just be `Resolver(self)`?

Reply:
changing, good point